starboard/linux/x64x11/toolchain/BUILD.gn

```diff
@@ -24,6 +24,7 @@ overridable_clang_toolchain("starboard") {
     clang_use_chrome_plugins = false
     enable_check_raw_ptr_fields = false
   }
+  propagates_configs = true
```

Comment:
Obviously only applies to this toolchain right now so other linux flavors will have issues

Reply:
Also , i think the files @y4vor is thinking of atleast in that ozone prototype were supposed to be built with chromium toolchain, this only applies to starboard toolchain. correct me if i'm wrong @y4vor

Reply:
talked offline, this makes sense. We are just propogating the public starboard configs to whatever targets depend on them . (Configs are private by default)